web/developer.actyx.com/docs/how-to/app-auth/generate-app-license.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Generate your own app license
+id: generate-app-license
+hide_title: false
+hide_table_of_contents: false
+image: /images/defaults/default.svg
+---
+
+This guide will walk you through the process of licensing an app.
+
+When running an Actyx node in production mode, your apps are required to be licensed.
+This mechanism provides a more fine grained control over which apps are _authorized_ to run on which nodes.
+
+To generate an app license, we use `ax-cert`:
+
+```bash
+$ cd rust/actyx
+$ cargo run --bin ax-cert -- app-license \
+    --actyx-private-key 0WBFFicIHbivRZXAlO7tPs7rCX6s7u2OIMJ2mx9nwg0w= \
+    --app-id com.example.app \
+    --email 'sample@actyx.com' \
+    --expires-at '2024-11-28T12:00:00Z'
+
+v25saWNlbnNlVmVyc2lvbgBrbGljZW5zZVR5cGWhaGV4cGlyaW5nomVhcHBJZG9jb20uZXhhbXBsZS5hcHBpZXhwaXJlc0F0dDIwMjQtMTEtMjhUMTI6MDA6MDBaaWNyZWF0ZWRBdHgbMjAyMy0wOS0yMlQxMzowMTowOS43Njg0ODNaaXNpZ25hdHVyZXhYdEJFK2FIalhmMDdBaG9mVXMwMFdIcjBDZDJiaFBJbmRSY0x0OU5oYk9VN3hpWGFvMENrY0MzU000MEtSRytMcmhtWDhXdlhUdGJuTzB3QzZESWNYQmc9PWlyZXF1ZXN0ZXKhZWVtYWlscHNhbXBsZUBhY3R5eC5jb23/
+```
+
+You can then use the license by setting it in your node settings:
+
+```json
+"licensing": {
+    "apps": {
+        "com.example.app": "v25saWNlbnNlVmVyc2lvbgBrbGljZW5zZVR5cGWhaGV4cGlyaW5nomVhcHBJZG9jb20uZXhhbXBsZS5hcHBpZXhwaXJlc0F0dDIwMjQtMTEtMjhUMTI6MDA6MDBaaWNyZWF0ZWRBdHgbMjAyMy0wOS0yMlQxMzowMTowOS43Njg0ODNaaXNpZ25hdHVyZXhYdEJFK2FIalhmMDdBaG9mVXMwMFdIcjBDZDJiaFBJbmRSY0x0OU5oYk9VN3hpWGFvMENrY0MzU000MEtSRytMcmhtWDhXdlhUdGJuTzB3QzZESWNYQmc9PWlyZXF1ZXN0ZXKhZWVtYWlscHNhbXBsZUBhY3R5eC5jb23"
+    }
+}
+```
```

Comment:
This should be accompanied by the documentation on how to create the signed app manifest that the `com.example.app` then needs to use.

Reply:
Isn't that covered in the "Sign app manifest" tutorial? Should I had a backlink to it?

Reply:
Yes, a link suffices.